src/Transient_Command.php

```diff
@@ -221,53 +223,166 @@ public function type() {
 
 	/**
 	 * Deletes all expired transients.
+	 *
+	 * Only deletes the expired transients from the database.
```

Comment:
This should behave the same as other commands and should delete site transients only when the  `--network` flag is set.